SwrveSDK/src/google/java/com/swrve/sdk/Swrve.java

```diff
@@ -128,8 +128,8 @@ public void setPushNotificationListener(ISwrvePushNotificationListener pushNotif
      * the Google Play Store or enable it in the device's system settings.
      */
     protected boolean checkPlayServices() {
-        int resultCode = GooglePlayServicesUtil.isGooglePlayServicesAvailable(context.get());
-        return resultCode == ConnectionResult.SUCCESS;
+        GoogleApiAvailability api = GoogleApiAvailability.getInstance();
+        return api.isGooglePlayServicesAvailable(context.get()) == ConnectionResult.SUCCESS;
```

Comment:
Can we send to logcat a message containing the ConnectionResult value if its not success? Also rename it to isGooglePlayServicesAvailable?